pkg/controller/model/GrafanaDatasourcesConfig.go

```diff
@@ -0,0 +1,32 @@
+package model
+
+import (
+	"github.com/integr8ly/grafana-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/grafana-operator/pkg/controller/config"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GrafanaDatasourcesConfig(cr *v1alpha1.Grafana) (*v1.ConfigMap, error) {
+	configMap := &v1.ConfigMap{}
+	configMap.ObjectMeta = v12.ObjectMeta{
+		Name:      config.GrafanaDatasourcesConfigMapName,
+		Namespace: cr.Namespace,
+	}
+
+	return configMap, nil
```

Comment:
I think you can remove the `error` from the return args list since you're always returning nil here